src/Chapter1.hs

```diff
@@ -641,7 +656,9 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+-- firstDigit n = if div n 10 == 0 then n else firstDigit (div n 10)
```

Comment:
Yeap, this is almost right. You only need to think of the case of the negative numbers. For that, you would need to use `abs` function 👍🏼